src/flow.jl

```diff
@@ -1382,12 +1384,16 @@ function initialize_floodplain_1d(nc, config, inds, riverwidth, riverlength, ind
         p[2:end, i] = 2.0 * p_unit[2:end, i] .* h
         p[2:end, i] = cumsum(p[2:end, i])
         a[:, i] = cumsum(a[:, i])
+
+        riv_cells += min(incorrect_vol, 1)
```

Comment:
This counter is not correctly keeping track of the river cells. A counter (or flag) should be reset at the start (but inside) the for loop, to correctly count the number of affected river pixels